var/spack/repos/builtin/packages/bml/package.py

```diff
@@ -7,20 +7,25 @@
 
 
 class Bml(CMakePackage):
-    """The basic matrix library (bml) is a collection of various matrix data
-    formats (in dense and sparse) and their associated algorithms for basic
-    matrix operations."""
+    """The basic matrix library (bml) is an implementation of various matrix data
+    formats (dense and sparse) and their associated algorithms for basic
+    matrix operations used in electronic structure solvers."""
 
     homepage = "https://lanl.github.io/bml/"
-    url = "https://github.com/lanl/bml/tarball/v1.2.2"
+    url = "https://github.com/lanl/bml/tarball/v2.2.0"
     git = "https://github.com/lanl/bml.git"
 
+    maintainers = ["jeanlucf22"]
+
     version("develop", branch="master")
+    version("2.2.0", sha256="9a975072263e7f7b0189c2544dc824f936dd736c16760a5b5d2de6a638da854f")
```

Comment:
Odd.  This still does not match what I'm getting.  How are you calculating the checksums?

```
    version("2.2.0", sha256="77e0a631203b84b09365a9a2a7d5269185799c60cf986d9242e7a315011a7d7f")
```

Reply:
sorry, the "url" was incorrect and I was ignoring the warning... I think I was getting random hashes...